classes/Tag.php

```diff
@@ -231,7 +231,7 @@ public static function getProductTags($idProduct)
         }
         $result = [];
         foreach ($tmp as $tag) {
-            $result[(int) $tag['id_lang']][] = $tag['name'];
+            $result[$tag['id_lang']][] = $tag['name'];
```

Comment:
reverted int casting to avoid changing legacy classes. (as now i still need to loop through when building DTO, so I cast there instead)